doc/en/user/source/community/gwc-mbtiles/index.rst

```diff
@@ -0,0 +1,50 @@
+.. _community_gwc_mbtiles:
+
+GWC MBTiles layer plugin
+========================
+
+This plugin allows to adding a GWC MBTiles Layer to GeoServer's cached Tile Layers configuration so that GeoServer can load cached tiles from ready-to-use MBTiles.
```

Comment:
```suggestion
This plugin allows adding a GWC MBTiles Layer to GeoServer's cached Tile Layers configuration so that GeoServer can load cached tiles from ready-to-use MBTiles.
```